CHANGELOG.adoc

```diff
@@ -112,6 +112,7 @@ This is now fixed
 - https://github.com/eclipse-sirius/sirius-web/issues/4261[#4261] [core] Enable image cache
 - https://github.com/eclipse-sirius/sirius-web/issues/4225[#4225] [emf] Add support for localized enumeration literals in EMF property forms
 - https://github.com/eclipse-sirius/sirius-web/issues/4227[#4227] [emf] Add support for custom numeric and boolean types in property forms
+- https://github.com/eclipse-sirius/sirius-web/issues/4317[#4317] [sirius-web] Export `NewObjectModal`, `NewRootObjectModal`, and `NewRepresentationModal`
```

Comment:
Please specify that these are frontend components, and the purpose of the export (to be used by Sirius Web based applications)